### DISCUSSION TITLE
Explanation for colspec numbers in manual

### DISCUSSION THREAD

**Discussion by mafsi** [lvjr/tabularray#205] [Q&A]
On `page 7` in manual, there is an example:

```latex
\begin{tblr}{width=0.8\linewidth,colspec={|X[2,l]|X[3,l]|X[-1,r]|X[r]|}}
\hline
Alpha & Beta & Gamma & Delta \\
\hline
\end{tblr}
```
What is `2,3`, and `-1` in `colspec`?

**Comment by lvjr** [comment]
The default coefficient is `1` for a `X` column, i.e., `X` means `X[1]`. For column spec `X[2]X[3]X[1]X[1]`, the width of columns will be `2:3:1:1`. But for column spec `X[2]X[3]X[-1]X[1]`, the width of columns will be firstly computed by `2:3:1:1`, then if the natural width of the third column is less than the colmputed width, it will be narrowed down to its natural width.

**Reply by mafsi** [reply]
Can you explain please the term `coefficient`? I always thought that those numbers are `cm`.

**Reply by lvjr** [reply]
it is a number, not a dimension.
